UPGRADE-14.0.md

```diff
@@ -836,7 +836,7 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
         ```diff
             public function create(
                 ProductData $productData
-        +       ?ProductRecalculationPriorityEnumInterface $priority = null,
+        +       string $priority = ProductRecalculationPriority::REGULAR,
```

Comment:
There are a lot of things missing in the upgrade notes

Reply:
actually all the changes are within the `14.0` version